_includes/styles.scss

```diff
@@ -814,3 +814,72 @@ html[data-current="contact"] main > form {
   width: 100%;
   margin-top: 8px;
 }
+
+/*fontawesome-menu styles*/
+
+.bg-primary{
+  color:white;
+  background-color: #337ab7;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.services {
+    padding: 10px 0;
+    border-radius: 6px;
+    margin-top: auto;
+    margin-bottom: auto;
+}
+
+.container-fw{
+  display: inline-block;
+  height: auto;
+}
+
+.text-vertical-center {
+    display: table-cell;
+    text-align: center;
+    vertical-align: middle;
+}
+
+.col-lg-10{
+  display: inline-block;
+  height: auto;
+}
+
+.row{
+  display: inline-flex;
```

Comment:
`display: inline-flex` cause the issue, it is ok for large display but not for the small one. . need to change the code.